### PR TITLE
[GOVCMSD8-143] Fixing the script to disallow all in robots.txt for no…

### PR DESCRIPTION
**Pull request by gargsuchi**
…n-prod environments.

### CODE REVIEW

**Comment by fubarhouse** [comment]
Before:
image

After:
image

**Comment by pandaskii** [comment]
Retest

**Comment by tobybellwood** [comment]
The test fails here because we actually now have a correct robots.txt in place
https://github.com/govCMS/govcms8lagoon/blob/master/tests/goss/sanitise.yaml#L27

the test line for robots.txt either needs to be rewritten to check for the correct string, or removed.

**Comment by pandaskii** [comment]
It is testing govcms8dev/govcms8:1.0.0-beta3?

**Comment by tobybellwood** [comment]
@pandaskii - nice pickup - branch needs rebasing on master...

File: tests/goss/sanitise.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ file:
   /app/web/web.config:
     exists: false
   /app/web/robots.txt:
-    exists: false
+    exists: true
```
Comment (tobybellwood) [inline tests/goss/sanitise.yaml]: the robots.txt file won't exist in the test environment - there is no "production" flag - we should remove this check for the time being.

**Comment by pandaskii** [comment]
Thanks for contributions

**Pull request merged**